docs/content/examples/developers.md

```diff
@@ -282,7 +252,7 @@ import (
 )
 ```
 
-*Note that `pub.Publish(e)` does not return an immediate error, it's an asynchronous operation so if we want to check for errors we have to do so after the fact. This means that we can't be sure which event actually triggered the error.*
+*Note that `client.Publish(topicID, e)` does not return an immediate error, it's an asynchronous operation so if we want to check for errors we have to do so after the fact. This means that we can't be sure which event actually triggered the error.*
```

Comment:
```suggestion
*Note that `client.Publish(topicID, e)` does not return an immediate error, it's an asynchronous operation so if we want to check if the event was actually published later on in the code we can use the `Acked()` or `Nacked()` methods on the event itself.*

```golang
acked, err := e.Acked()
if err != nil {
    fmt.Printf("could not check ack status for event: %s\n", err)
}

if acked {
    fmt.Println("event was committed!")
}

nacked, err := e.Nacked()
if nacked {
    fmt.Printf("event failed to commit with error: %s\n", err)
}
```
```